app/controllers/api/v2/courses_controller.rb

```diff
@@ -167,7 +167,8 @@ def course_params
           .permit(
             :english,
             :maths,
-            :science
+            :science,
+            :qualification
```

Comment:
whats the diff between `qualification` vs `qualifications`

Reply:
A subtle one. I mentioned it a bit more in the frontend PR:

> Removes the only frontend dependency on `course.qualifications` (`['qts', 'pgce']`), favouring `course.qualification` (`pgce_with_qts`).

Qualifications is an array.
Qualification is a string that combines the array.

Frontend will set the string, and use the string, and doesn't need the array – we store as array in case we need to do different things for QTS/PGCE in the future.